mysqli/mysqli.php

```diff
@@ -549,7 +552,7 @@ public function query ($query, $resultmode = MYSQLI_STORE_RESULT) {}
 	 * </p>
 	 * @return bool true on success or false on failure.
 	 */
-	public function real_connect ($host = null, $username = null, $passwd = null, $dbname = null, $port = null, $socket = null, $flags = null) {}
+	public function real_connect ($host = null, $username = null, $passwd = null, $dbname = null, $port = 0, $socket = null, $flags = null) {}
```

Comment:
according to https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.stub.php#L174 stub is
```
public function real_connect(
        ?string $host = null,
        ?string $user = null,
        ?string $password = null,
        ?string $database = null,
        ?int $port = null,
        ?string $socket = null,
        int $flags = 0
    ) {}
```
So I think default values should be updated accordingly